static/js/publisher/form/AccordionHelp.js

```diff
@@ -33,7 +33,7 @@ class AccordionHelp extends React.Component {
 
     return (
       <Fragment>
-        <p className="p-form-help-text u-no-margin">
+        <p className={`p-form-help-text${open ? " u-no-margin--bottom" : ""}`}>
```

Comment:
I hate writing these and hate reading these :) Some day we need to finally get some nice classname utility function.

On that note, I think its unnecessary optimisation that makes it harder to read: 
```suggestion
        <p className={`p-form-help-text ${open ? "u-no-margin--bottom" : ""}`}>
```

Reply:
Once we're happy with the other stuff, I'll accept this change before merging!